app/routes.py

```diff
@@ -1,2 +1,85 @@
-from flask import Blueprint
+from app import db
+from flask import Blueprint, jsonify, make_response, request
+from app.models.planet import Planet
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+def make_input_valid(number):
+    try:
+        int(number)
+    except:
+        return make_response(f"{number} is not an int!", 400)
+
+def is_parameter_valid(parameter_id):
+    if make_input_valid(parameter_id) is not None:
+        return make_input_valid(parameter_id)
+    elif Planet.query.get(parameter_id) is None:
+        return make_response(f"{parameter_id} is not a valid id!", 404)
+
+@planets_bp.route("", methods=["GET"])
+def read_all_planets():
+    planets = Planet.query.all()
+    planet_list = []
+    for planet in planets:
+        planet_list.append(planet.make_dict())
+    return jsonify(planet_list)
+
+
+@planets_bp.route("", methods=["POST"])
+def post_new_planet():
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"], 
+                        description=request_body["description"],
+                        matter=request_body["matter"])    
```

Comment:
Here's a suggestion for some input handling. If the request body is missing one of the keys, this code will raise a KeyError exception. Consider handling this way:

```suggestion
    if "name" not in request_body or "description" not in request_body or "matter" not in request_body:
        return {"error": "incomplete request body"}, 400

    new_planet = Planet(name=request_body["name"], 
                        description=request_body["description"],
                        matter=request_body["matter"])    
```